### ISSUE TITLE
Jungheinrich ETV 216I

### ISSUE THREAD

**Issue by ChrisKeng** [900hasse/FS22_REAwheels#13]
Hello,

with activated REA Wheels, it is not possibile to drive the Jungheinrich ETV 216I from Update 1.4.1.

Max Speed will be 1-2 km/h, if REA Wheels is deactivated, it works fine.

**Comment by Kotlett** [comment]
Yes, also some other Vehicles with tiny wheels have this or similar problems with the mod. Increasing the affected wheel-size in the lua doesn't seem to change anything.

Is there some way to exclude particular vehicles from the mod? This could be an easy fix for such problems...
